msm/src/verifier.rs

```diff
@@ -35,7 +35,7 @@ pub fn verify<
 >(
     domain: EvaluationDomains<G::ScalarField>,
     srs: &OpeningProof::SRS,
-    constraint_exprs: &Vec<E<G::ScalarField>>,
+    constraints: &Vec<E<G::ScalarField>>,
```

Comment:
Just renamed it for uniformity because in the prover algorithm they were also called `constraints`